eng/common/scripts/Package-Properties.ps1

```diff
@@ -266,6 +270,8 @@ function Get-PrPkgProperties([string]$InputDiffJson) {
             # will apply to all packages that exist in that directory.
             $triggeringCIYmls = $triggeringPaths | Where-Object { $_ -like "*ci*.yml" }
 
+            # what would happen if someone included a ci.yml in the triggering paths?
```

Comment:
It's already always there implicitly due to `triggeringPaths` artifact update in `InitializeCIArtifactDetails`.

Reply:
I mean... What happens if, for some reason, eventhubs/ci.yml includes:
```
      triggeringPaths:
      - /sdk/storage/ci.yml
```

Do we make assumptions about ci*.yml triggeringPaths that would be wrong if someone made an explicit, cross service reference to a ci*.yml file?